dashboard_viewer/dashboard_viewer/settings.py

```diff
@@ -275,6 +276,21 @@
         "If a Data Source owner can change the draft status when editing its details",
         bool,
     ),
+    "SUPERSET_HOST": (
+        "https://superset.ehden.eu",
+        "Host of the target superset installation. Used to redirect to the dashboard of a database",
+        str,
+    ),
+    "DATABASE_DASHBOARD_IDENTIFIER": (
+        "database-level-dashboard",
+        "Identifier of the database dashboard on the Superset installation.",
+        str,
+    ),
+    "DATABASE_FILTER_ID": (
+        69,
```

Comment:
@aspedrosa why 69?

Reply:
This is the id of the Datasource filter chart present on our Database-Level Dashboard.